README.md

```diff
@@ -168,40 +164,67 @@ Authentication concerns (your typical `Devise` configuration) are handled by the
 
 ### Webpacker, the asset pipeline, and ES6
 
-By default ES6 will not work for files in `app/assets/javascript` since Uglifier will fail to
-process them. This is why we applied the following change to `config/production.rb`, which allows
-you to use ES6 project wide:
+By default ES6 will not work for files in `app/assets/javascript` since Uglifier will fail to process them. This is why we applied the following change to `config/production.rb`, which allows you to use ES6 project wide:
 
 ```diff
 -  config.assets.js_compressor = :uglifier
 +  config.assets.js_compressor = Uglifier.new(harmony: true)
 ```
 
-Also note that for everything to work properly on Heroku, you need to set up your buildpacks like
-this:
+Also note that for everything to work properly on Heroku, you need to set up your buildpacks like this:
 
 ```
 heroku buildpacks:clear
 heroku buildpacks:set heroku/nodejs
 heroku buildpacks:add heroku/ruby
 ```
 
+### TypeScript
+
+TypeScript is supported out of the box. See the example in `app/javascript`.
```

Comment:
Did we remove the example this refers to?

Reply:
well that example is TS now, would you prefer to change it back?